Set/Card/CardView.swift

```diff
@@ -16,24 +16,31 @@ struct CardView: View {
         GeometryReader { geometry in
             ZStack {
                 let shape = RoundedRectangle(cornerRadius: Constants.cornerRadius)
-                shape.fill(cardViewModel.backroundColor)
-                shape.strokeBorder(lineWidth: Constants.borderWidth)
-                    .foregroundColor(cardViewModel.strokeColor)
+                if cardViewModel.cardIsFaceUp {
```

Comment:
I think that rendering concrete state if card, `isCardFaceUp` (better than `cardIsFaceUp`, boolean should start with `is`) can be moved to standalone `FaceCardView` and `BackCardView` that just accept in `init` some attributes to render correctly

Reply:
fixed